doc/gov/chapters/chapter02.md

```diff
@@ -5,43 +5,53 @@
 
 ## Table of Contents
 * [2.1 Introduction](#2.1)
-* [2.2 Request Membership](#2.2)
+* [2.2 Request Participation](#2.2)
 * [2.3 Prerequisites](#2.3)
-* [2.4 Beginning Engagement](#2.4)
-* [2.5 Member Administration](#2.4)
+* [2.4 Engagement](#2.4)
+* [2.5 Participant and Contributor Administration](#2.4)
 
 <a name="2.1"></a>
 ## 2.1 Introduction
 
-As an open-source community organization, CNTT has developed the following process to ensure delivery on CNTT objectives. This process has several goals. First, inform new members of the resources available upon joining the community. Second, communicate the organizational and legal structures upon which the community operates. Help new members become fully engage & contributing effectively to the work streams.
+As an open-source community organization, CNTT has developed the following process to ensure delivery on CNTT objectives. This process has several goals. First, inform new participants of the resources available upon joining the community. Second, communicate the organizational and legal structures upon which the community operates. Help new members become fully engage & contributing effectively to the work streams.
```

Comment:
In this document, is participants intended to refer to individuals, or companies?

Reply:
either